utils/githooks/pre-commit.d/20-codespell.sh

```diff
@@ -22,4 +23,4 @@ fi
 
 echo "Checking for spelling mistakes"
 # Convert file names to relative path format that codespell expects. I.e. "./path"
-_git_diff_cached_files | xargs -r -n 1 -I% echo "./%" | xargs -r codespell
+_git_diff_cached_files | xargs -r -n 1 -I% echo "./%" | xargs -r codespell --skip ./src/control/vendor,./src/control/go.sum,./utils/*.patch
```

Comment:
These are actually specified here:
https://github.com/daos-stack/daos/blob/ae9f129624c784a856e46a92b453944ed6b8f56a/pyproject.toml#L7-L11

Reply:
Thanks, I will change it